tools/mapping/gen_domain_files/src/gen_domain.F90

```diff
@@ -38,20 +38,26 @@ program fmain
   character(LEN=512) :: usercomment ! user comment
   character(LEN= 8)  :: cdate       ! wall clock date
   character(LEN=10)  :: ctime       ! wall clock time
+  real(r8) :: fminval = 0.001_r8    ! min allowable land fraction; frac set to 0 if frac < fminval
```

Comment:
It seems like the convention is to declare variables here and set default values below (e.g. how `set_fv_pole_yc` is handled)

Reply:
I can change this. I think I put these here because originally they were hard-coded as magic numbers elsewhere in the code and my first attempt was to just change them to parameter values here.

Reply:
This should be fixed (also moved `set_omask` initialization for consistency).